functional-tests/http/src/test/java/org/mule/test/http/functional/requester/streaming/HttpStreamingRequestNtlmProxyConfigTestCase.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.http.functional.requester.streaming;
+
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.OK;
+import static org.mule.runtime.http.api.HttpHeaders.Names.CONNECTION;
+import static org.mule.runtime.http.api.HttpHeaders.Names.PROXY_AUTHENTICATE;
+import static org.mule.runtime.http.api.HttpHeaders.Names.PROXY_AUTHORIZATION;
+import static org.mule.runtime.http.api.HttpHeaders.Values.KEEP_ALIVE;
+import static org.mule.test.http.functional.AllureConstants.HttpFeature.HttpStory.NTLM;
+import static org.mule.test.http.functional.AllureConstants.HttpFeature.HttpStory.PROXY;
+import static org.mule.test.http.functional.AllureConstants.HttpFeature.HttpStory.STREAMING;
+import static org.mule.test.http.functional.fips.DefaultTestConfiguration.isFipsTesting;
+
+import static javax.servlet.http.HttpServletResponse.SC_PROXY_AUTHENTICATION_REQUIRED;
+
+import static org.apache.http.entity.ContentType.TEXT_PLAIN;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assume.assumeFalse;
+
+import org.mule.tck.junit4.rule.DynamicPort;
+import org.mule.test.http.functional.requester.ntlm.AbstractAuthNtlmTestCase;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Stories;
+import io.qameta.allure.Story;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.client.fluent.Response;
+import org.apache.http.entity.StringEntity;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+
+
+@Stories({@Story(PROXY), @Story(NTLM), @Story(STREAMING)})
+public class HttpStreamingRequestNtlmProxyConfigTestCase
```

Comment:
This test caught my attention because it's testing the proxy authentication in a strange way... I see that you copied it from an older test, so it's not your fault, but I will ask you to add a comment here so we know what is it doing in the future. The comment should be something like:
The server that is working as a "proxy" (and executing the authentication) here is the one we usually use as the "target", but it works for the test purposes.